src/templates/react/Content/src/App.fs

```diff
@@ -0,0 +1,149 @@
+namespace FableElmishReactTemplate
+
+open Elmish
+open Elmish.Browser.Navigation
+open Elmish.Browser.UrlParser
+open Fable.Core
+open Fable.Core.JsInterop
+open Fable.Import
+open Fable.Import.Browser
+open FableElmishReactTemplate.Common
+
+module App =
+
+  importAll "../sass/main.sass"
+
+  type SubModels =
+    {
+      Home: Home.Model
+      Counter: Counter.Model
+    }
+
+    static member Initial =
+      {
+        Home = ""
+        Counter = 0
+      }
+
+  type Model =
+    {
+      CurrentPage: Page
+      SubModels: SubModels
+    }
+
+    static member Initial =
+      {
+        CurrentPage = Home
+        SubModels = SubModels.Initial
+      }
+
+  let urlUpdate (result: Option<Page>) model =
+    match result with
+    | None ->
+      Browser.console.error("Error parsing url")
+      ( model, Navigation.modifyUrl (toHash model.CurrentPage) )
+    | Some page ->
+        { model with CurrentPage = page }, []
+
+  let init result =
+    urlUpdate result Model.Initial
+
+  type Msg =
+    | CounterMsg of Counter.Msg
+    | HomeMsg of Home.Msg
+    | NavigateTo of Page
```

Comment:
Page navigation is already handled as part of the navigable app, we should guide the user towards relying on those facilities instead of burdening `update` function with it.